datascience/tables.py

```diff
@@ -2638,9 +2638,15 @@ def prepare_hist_with_group(group):
         values_dict = collections.OrderedDict(values_dict)
         if left_end is not None or right_end is not None:
             if left_end is None:
-                left_end = min([min(self.column(k)) for k in self.labels if np.issubdtype(self.column(k).dtype, np.number)])
+                if bins[0]:
```

Comment:
These new checks are for making sure that if the user specifies bins, then the histogram shading goes all the way to the edge of the first/last bin